src/pages/state/[state].content.yml

```diff
@@ -81,43 +84,51 @@ videos:
     title: Rep. Katie Porter at House Oversight Committee Hearing
     tags:
       - CA
+    description: ""
   - title: Patient advocates including Alonzo Mourning, Samantha Bee, and Jenna Zak
     link: https://twitter.com/oversightdems/status/1388204146969427971?s=10
     tags:
       - AL
 notes:
   - note: Passing (Tier 1) [in
       2018](https://www.federalregister.gov/documents/2020/12/02/2020-26329/medicare-and-medicaid-programs-organ-procurement-organizations-conditions-for-coverage-revisions-to)
+    voicesforreform: false
```

Comment:
Oh, interesting. Almost like a weird Netlify-CMS migration...

Reply:
Yes, just wanted to ensure any logic on it wouldn't break things, especially after I set required: true.